server/server.go

```diff
@@ -400,6 +400,9 @@ func (s *Server) onConn(conn *clientConn) {
 	if plugin.IsEnable(plugin.Audit) {
 		sessionVars.ConnectionInfo = conn.connectInfo()
 	}
+	type vars struct {
+		killed *uint32
+	}
```

Comment:
These lines are unnecessary, so how about removing them?

Reply:
> These lines are unnecessary, so how about removing them?

ok, I will remove in my next PR.